Remove Clarion shields

flappybatpal opened the pull request:
<!-- The text between the arrows are comments - they won't be visible on your PR. -->
<!-- To label this PR, add the label(s) without the prefixes surrounded by brackets anywhere, for example: [LABEL] -->
<!-- PRs should at least have one area (A-) label and at least one category (C-) label -->
[REMOVAL][QOL]

## About the PR <!-- Describe the Pull Request here. What does it change? What other things could this impact? -->
Removes the /area/station/shield_zone and /obj/machinery/shield_generator from Clarion.

## Why's this needed? <!-- Describe why you think this should be added to the game. -->
For a long time I have felt like the shield only gets in the way and provides no real gameplay benefit.
Today I observed it blocking both a devastating artifact bomb and the entrance to the escape shuttle and decided that was enough.

## Changelog <!-- If necessary, put your changelog entry below. Otherwise, /please/ delete this entire section. -->
<!-- Put how you want to be credited in the changelog in place of CodeDude. -->
<!-- Use (*) for major changes and (+) for minor changes. See the contributor guide for details. For example: -->

```changelog
(u)Flappybat
(*)Removed the shield and shield generator console from Clarion.
```

Sord213 reviewed (changes requested):
Could you fill the old console space with....something, and maybe delete the data terminal + wiring leading to it?

flappybatpal commented:
![image](https://user-images.githubusercontent.com/70909958/210063913-b696cd85-60d5-43fc-92e0-a878bc25f574.png)
Deleted terminal and cable.
Moved the fire locker that was awkwardly in front of a computer. Moved the chair over to the computer.
